Engine/GameEngine.cs

```diff
@@ -0,0 +1,410 @@
+using Engine.Helpers;
+
+namespace Engine;
+
+public static class GameEngine
+{
+    // Constants
+    public const int PlayersPerGame = 2;
```

Comment:
Is this actually hardcoded to two?

Reply:
Yea, it's just a two player game but wasn't sure if I was going to play around with more down the line